src/ComposerRequireChecker/ASTLocator/LocateASTFromFiles.php

```diff
@@ -25,9 +25,7 @@ public function __construct(Parser $parser, ?ErrorHandler $errorHandler)
     }
 
     /**
-     * @param Traversable|string[] $files
-     *
-     * @return Traversable|array[] a series of AST roots, one for each given file
+     * @return Traversable a series of AST roots, one for each given file
```

Comment:
We should document the `Traversable` input and output parameters - leaving them as just `Traversable` is not acceptable anymore